content/guides/latency.md

```diff
@@ -0,0 +1,130 @@
+
+\[Interactive Graphic\]
+
+#### Short-Lived
+
+Serverless functions may be executing independently in short-lived environments, it’s not feasible to have a long-running connection. This means every independent execution of a function must establish a connection to the database.
+
+\[Interactive Graphic\]
+
+**Tips for short-lived (serverless):**
+
+1. Execute functions in a region that is the same or close to your database
+2. Use an HTTP API \[Link to serverless driver\] \- establishing the HTTP connection is faster than establishing a TCP connection.
+3. Execute multiple queries in a single function \- so that you only pay the connection tax once.
+
+<Admonition type="tip" title="Reducing roundtrips with the Serverless Driver">
+Neon's [serverless driver](/docs/serverless-driver) and proxy have been optimized to reduce the number of roundtrips to the absolute minimum: Four round-trips to get a result.
+![Serverless driver round-trips](https://neon.tech/_next/image?url=https%3A%2F%2Fneondatabase.wpengine.com%2Fwp-content%2Fuploads%2F2023%2F08%2FDevDays-GM-Quicker-2.jpg&w=3840&q=85&dpl=dpl_93LwG65BEFC73DGwgsAif219AFEd)
+
+To read more about how this works, see: [Quicker serverless Postgres connections](https://neon.tech/blog/quicker-serverless-postgres)
+</Admonition>
+
+### Putting app and database close together
+
+I
+
+1. **App and database on same VM** - 1ms to connect and 1ms to query \[Not possible with Neon\] When you put your database and your app on the exact same machine, latency is measured in microseconds.
+2. **App and database in same region (datacenter) connecting over private network \-** 1ms to connect, 1ms to query \[Not exactly possible on Neon, you can connect through PrivateLink but it doesnt save you any time\]
+3. **App and database in same region (datacenter) connecting over public network** \- 3ms to connect, 1ms to query \- This is the fastest way to connect that is accessible to all Neon users. What you’re doing here is picking the same AWS or Azure region for your database and your application (be it serverless, like lambda functions on AWS, or serverful).
+   1. You still use the publicly routable address of your database, but data is traveling 0 miles so it is very fast.
+   2. This doesn’t mean your app has to be running on AWS directly, for example Vercel runs infra on AWS so picking the same Vercel and Neon region has the same effect.
+4. **App and database in different region \-** Now we get into that compounding effect
+
+### Minimizing time spent answering the query
+
+Until you start to scale, the vast majority of your standard database queries are going to execute in single digit milliseconds.
+
+[Docs Tips]
+
+## Definitions
+
+### Serverless
+
+[Docs serverless terminology]
```

Comment:
Our definition in the docs is minimal: https://neon.tech/docs/reference/glossary#serverless
This feature guide goes into more more about "what serverless means to us":
https://neon.tech/docs/introduction/serverless
I could revise our glossary term based on "what serverless means to us"